src/script/interpreter.cpp

```diff
@@ -303,16 +303,18 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
 
     try
     {
-        while (pc < pend)
+        while (pc.size() > 0)
         {
             bool fExec = !count(vfExec.begin(), vfExec.end(), false);
 
             //
             // Read instruction
             //
-            if (!script.GetOp(pc, opcode, vchPushValue))
+            Span<const unsigned char> push_value;
+            if (!GetScriptOp(pc, opcode, &push_value)) {
```

Comment:
nit: Braces can be removed here as the if-body only consists of one single statement (to be consistent with the remaining ifs in the same function).

Reply:
I'd rather not. The style guide only allows brace-less ifs if the then clause is on the same line as the if condition. Doing so would make the line a bit long in my opinion.

The fact that it's inconsistent with the code around it doesn't matter.

Reply:
Oh, my assumption was that mimicing the surrounding style was favored over new style rather than the other way around, mea culpa.